spec/text2pngSpec.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+const text2png = require('../index.js');
+const looksSame = require('looks-same');
+
+function renderSass(file, prefix) {
```

Comment:
Since the test worked, I merged it, but what is the renderSass function doing?

Reply:
Oh, we should get rid of that. I copied it from another project I was working on